### PR TITLE
Factorygirl multiples

### PR DESCRIPTION
**Pull request by dan-palmer**
- [x] destination multiples on move
- [x] medication multiples on healthcare
- [x] current offence multiples on offences
- [x] past offence multiples on offences
- [x] hook up to escort

### CODE REVIEW

File: spec/factories/medication_factory.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :medication do
+    description { Faker::Beer.name }
```
Comment (whoojemaflip) [inline spec/factories/medication_factory.rb:3]: 🐝 🍺 :bear:

File: spec/factories/offences_factory.rb [hunk 2]
```diff
@@ -3,6 +3,11 @@
     release_date '15/09/2027'
     workflow_status 'complete'
 
+    current_offences { build_list :current_offence, 2 }
```
Comment (whoojemaflip) [inline spec/factories/offences_factory.rb]: `rand(5)` current_offences?

Comment (dan-palmer) [inline spec/factories/offences_factory.rb]: 👍 we can apply this to all of the lists

**Comment by whoojemaflip** [comment]
💪

**Pull request merged**